cli/cli/commands/gateway/gateway.go

```diff
@@ -41,12 +35,13 @@ func run(cmd *cobra.Command, args []string) error {
 		return stacktrace.Propagate(err, "Expected to be able to get a Kurtosis backend connected to the cluster, instead a non-nil error was returned")
 	}
 
-	// TODO Store kube config path in configuration and read from there
-	kubeConfigPath := path.Join(os.Getenv("HOME"), ".kube", "config")
-
-	kubernetesConfig, err := clientcmd.BuildConfigFromFlags(emptyConfigMasterUrl, kubeConfigPath)
+	// TODO: get this from the backend's k8s manager somehow?
+	kubernetesConfig, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		clientcmd.NewDefaultClientConfigLoadingRules(), 
+		&clientcmd.ConfigOverrides{},
+	).ClientConfig()
```

Comment:
we have a lint rule called exhaust struct that requires filling out every field in a struct - if you add these with nils/empties CI should build!